histoqc/LightDarkModule.py

```diff
@@ -55,24 +55,14 @@ def getIntensityThresholdPercent(s, params):
     name = params.get("name", "classTask")
     logging.info(f"{s['filename']} - \tLightDarkModule.getIntensityThresholdPercent:\t {name}")
 
-    lower_thresh = float(params.get("lower_threshold", -float("inf")))
-    upper_thresh = float(params.get("upper_threshold", float("inf")))
+    lower_thresh = float(params.get("lower_threshold", "-inf"))
+    upper_thresh = float(params.get("upper_threshold", "inf"))
 
     # Prepare parameter names due to issues #213 and #219
     # set lower standard deviation
-    if params.get('lower_var'):
-        lower_std = float(params.get('lower_var'))
-    elif params.get('lower_std'):
-        lower_std = float(params.get('lower_std'))
-    else:
-        lower_std = float("inf")
+    lower_std = float(args.get("lower_var") or args.get("lower_std") or "-inf")
```

Comment:
wOoOo! super elegant. i had *no* idea that you could this in python (i actually fired up my interpreter to play with it). i had assumed it would turn simply a true/false value after evaluating the condition, but it actually returns the first non 0/None value? amazing!